apps/routes/test/repo_test.exs

```diff
@@ -1,5 +1,5 @@
 defmodule Routes.RepoTest do
-  use ExUnit.Case, async: true
+  use ExUnit.Case, async: false
```

Comment:
Looks like this might have caused a test failure in `SiteWeb.ScheduleController.AlertsControllerTest`

Reply:
Reverted this and still see these failures; I think I broke these elsewhere. Will address outside this PR.